### ISSUE TITLE
Stop maintaining this fork

### ISSUE THREAD

**Issue by WinterMute** [cotodevel/dswifi#1]
Please submit your changes upstream, maintaining your own fork with it's own version numbers doesn't help anyone, least of all yourself.

There was a reason why I asked you to start with the spinlock changes & showed you what the diff should look like.

**Comment by cotodevel** [comment]
For what? For being told that i'm commiting gits the wrong way? Doesn't go towards to you, you have helpful. But having messages like: https://github.com/devkitPro/dswifi/pull/2 . I don't have time for that.

**Comment by asiekierka** [comment]
@cotodevel That's when you do two commits (the spinlock.s push, and the cleanup). Even if just for the purposes of another human being reviewing your code, that's an important thing to do - you cannot be the only person in the world who understands the changes, and you must perform the act of explaining those changes to others.

Also, you said "forks", but you did not use the Fork button. That's horrifying, and the only possible explanation for it is "I didn't want anyone else to discover my library" - which is against any semblance of a cooperative community spirit.

**Comment by WinterMute** [comment]
well, if you'd actually done what you were asked to do with the spinlock change then it would already be committed upstream.

The second pull request was better from a whitespace perspective. Unfortunately you chose to commit too much in one go with meaningless commit messages.

The nesDS nifi looks like a completely separate issue - I'd need to see the code but it looks like you're attempting to compile nesDS with dswifi included in it's source tree which is pretty much guaranteed to cause problems.

It would be much easier for everyone if you contributed your changes upstream in a way that they're going to be accepted.

Have a read of this - https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/

**Comment by cotodevel** [comment]
No. NesDS's DSWIFI source is not the only one spitting the duplicate errors. SnemulDS does the same.
Also NesDS is *already* merged with modified DSWIFI 0.3.20 and NiFI will work just fine. 

I basically want to add DSNIFI as submodule for NesDS, but that requires a script to be run to fetch DSWIFI along NesDS, I just merged all together. And I assure it works.

I have set up a custom interrupt handler for other project, and re-use DSWIFI and replaced stock libnds FIFO overlay, so I have WIFI-ability on that project as well. But that is not for release. And it will work just fine as well.
